CHANGELOG.md

```diff
@@ -40,6 +40,8 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 ### New
 
+- **Azure Service Bus**: Add support for Shared Access Signature (SAS) tokens for authentication. ([#2920](https://github.com/kedacore/keda/issues/2920))
```

Comment:
This line is duplicated. Remove it please , in fact, `General:` section is before scalers